ts/sagas/backendInfo.ts

```diff
@@ -18,7 +18,7 @@ import { startTimer } from "../utils/timer";
 // load backend info every hour
 const BACKEND_INFO_LOAD_INTERVAL = 60 * 60 * 1000;
 
-// retry loading backend info every 10 seconds on error
+// retry loading backend info every 10 hours on error
```

Comment:
keep it shortly since we need this information to avoid the access to some functionalities
A reasonable value could be **5 seconds**

Reply:
Could you check this point?

Reply:
@Undermaken I set it like this:
BACKEND_INFO_LOAD_INTERVAL = 1 hour
BACKEND_INFO_RETRY_INTERVAL (on error) = 5 seconds
ok?

Reply:
I just updated this section.

```typescript
// load backend info every hour
const BACKEND_INFO_LOAD_INTERVAL = 60 * 60 * 1000;

// retry loading backend info every 10 seconds on error
const BACKEND_INFO_RETRY_INTERVAL = 10 * 1000;
```